osmaxx-py/osmaxx/excerptexport/rest_api/serializers.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework_gis import serializers as gis_serializers
+from rest_framework_gis.fields import GeometrySerializerMethodField
+
+from osmaxx.excerptexport.models import Excerpt
+from osmaxx.excerptexport.models.bounding_geometry import BoundingGeometry
+from osmaxx.utilities.shortcuts import get_actual
+
+
+class BoundingGeometrySerializer(gis_serializers.GeoFeatureModelSerializer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    geom = GeometrySerializerMethodField()
```

Comment:
could rename it geometry now, since that name is no longer being used by any other class. Other propositions?